_i18n/es/resources/user-guides/prove-payment.md

```diff
@@ -1,86 +1,78 @@
 
-Note that this will only work if monero-wallet-cli is set to save transaction keys. To double check:
+Tenga en cuenta que esto sólo funcionará si monero-wallet-cli está configurado para guardar llaves de transacción. Para revisar doblemente:
 
 > set
 
-If it's set to 0, set it to 1:
+Si set es 0, cámbialo a 1:
 
 > set store-tx-info 1
 
 #### GUI
 
-Alice can open her monero-wallet-gui and go to the history page to see her transaction detail:
+Alicia puede abrir monero-wallet-gui e ir al historial para ver los detalles de su transacción:
 
-![History](png/prove-payment/history.png)
+![Historial](png/prove-payment/history.png)
 
-Here, she can copy the transaction ID and Bob's address by clicking on each of them.
-Then she can click on `P` to get a payment proof (transaction key):
+Aquí, ella puede copiar el ID de transacción y la dirección de Bob haciendo clic en cada uno. Entonces ella puede hacer clic en `P` para obtener una prueba de pago (llave de transacción):
 
-![Payment proof](png/prove-payment/payment-proof.png)
+![Prueba de pago](png/prove-payment/payment-proof.png)
 
 
 ---
 
-Alice can now send Charlie the transaction key along with transaction ID and Bob's address.
+Alicia ahora puede enviarle a Charlie la llave de transacción junto al ID de transacción y la dirección de Bob.
 
-Note: if several transactions were made, this needs repeating for each such transaction.
+Nota: si varias transacciones fueron hechas, esto debe repetirse para cada transacción.
 
-### Check payments
+### Revisar pagos
 
-Charlie now received those three pieces of information, and wants to check Alice is telling the truth: on an up to date
-blockchain,
+Charlie ahora recibió esas tres piezas de información, y quiere revisar si Alicia está diciendo la verdad: en una blockchain actualizada,
 
 #### CLI
 
-Charlie types in monero-wallet-cli:
+Charlie escribe en monero-wallet-cli:
 
 > check_tx_key TXID TXKEY ADDRESS
 
-The information supplied by Alice plugs neatly instead of the placeholders. monero-wallet-cli will use the transaction
-key to decode the transaction, and display how much this particular transaction sent to this address. Obviously,
-Charlie will want to double check with Bob the address is really his - same as with Bitcoin.
+La información provista por Alicia se coloca pulcramente en lugar de los marcadores. monero-wallet-cli usará la llave de transacción para decodificar la transacción, y mostrar cuánto envió en particular la transacción a la dirección. Obviamente, Charlie querrá revisar doblemente con Bob que la dirección es realmente de él - al igual que con Bitcoin.
 
 #### GUI
 
-Charlie will open his monero-wallet-gui and go to the Advanced > Prove/Check page to fill the Check section with the informations provided by Alice:
+Charlie abrirá su monero-wallet-gui e irá a la página en Advanced > Prove/Check para llenar la sección Check con la información provista por Alicia:
```

Comment:
For the `Advanced > Prove/Check` and `Check`. Inside the `monero-wallet-gui` those words are already translated, so the guides can have those commands updated in the [monero-wallet-gui repository](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts). Maybe you didn't know this, so I'm sharing it with you. So I think we can work it out with the commands inside the translations files. I just remembered that the CLI and the GUI are already translated.

Reply:
Just a simple "find in page" will solve this, thanks.

Reply:
Just to remember that the CLI has not been translated to "es". Sorry for the misleading info.

Reply:
No worries, maybe we can work on it in a soon future :)